sdk-extensions/autoconfigure/build.gradle.kts

```diff
@@ -78,7 +78,6 @@ testing {
             environment("OTEL_PROPAGATORS", "tracecontext,baggage,b3,b3multi,jaeger,ottrace,test")
             environment("OTEL_EXPORTER_OTLP_HEADERS", "cat=meow,dog=bark")
             environment("OTEL_EXPORTER_OTLP_TIMEOUT", "5000")
-            environment("OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT", "2")
```

Comment:
This was interfering with the changes I was making to `FullConfigTest`. Setting span limits is well tested elsewhere, and we should limit the integration-style `FullConfigTest` to concepts that really require everything to be stood up in an end-to-end fashion.